py/server/deephaven/agg.py

```diff
@@ -201,35 +204,43 @@ def max_(cols: Union[str, List[str]] = None) -> Aggregation:
     return Aggregation(j_agg_spec=_JAggSpec.max(), cols=cols)
 
 
-def median(cols: Union[str, List[str]] = None) -> Aggregation:
-    """Create a Median aggregation.
+def median(cols: Union[str, List[str]] = None, average_evenly_divided: bool = True) -> Aggregation:
+    """Creates a Median aggregation which computes the median value within an aggregation group for each of the
+    given columns.
 
     Args:
         cols (Union[str, List[str]]): the column(s) to aggregate on, can be renaming expressions, i.e. "new_col = col";
             default is None, only valid when used in Table agg_all_by operation
+        average_evenly_divided (bool): when the group size is an even number, whether to average the two middle values
+            for the output value, default is True. when set to False, use the smaller value. This flag is only valid for
+            numeric types.
 
     Returns:
         an aggregation
     """
-    return Aggregation(j_agg_spec=_JAggSpec.median(), cols=cols)
+    return Aggregation(j_agg_spec=_JAggSpec.median(average_evenly_divided), cols=cols)
 
 
-def pct(percentile: float, cols: Union[str, List[str]] = None) -> Aggregation:
-    """Create a Percentile aggregation.
+def pct(percentile: float, cols: Union[str, List[str]] = None, average_evenly_divided: bool = False) -> Aggregation:
+    """Creates a Percentile aggregation which computes the percentile value within an aggregation group for each of
+    the given columns.
 
     Args:
         percentile (float): the percentile used for calculation
         cols (Union[str, List[str]]): the column(s) to aggregate on, can be renaming expressions, i.e. "new_col = col";
             default is None, only valid when used in Table agg_all_by operation
+        average_evenly_divided (bool): when the percentile splits the group into two halves, whether to average the two
+            middle values for the output value, default is False, meaning to use the smaller value. This flag is only
+            valid for numeric types.
```

Comment:
See other suggestion.  These are worded in a way that is complex syntactically and users may have a hard time following.